data/json/recipes/recipe_others.json

```diff
@@ -146,6 +146,25 @@
   ]
 },{
   "type" : "recipe",
+  "result": "felt_patch",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_OTHER",
+  "skill_used": "fabrication",
+  "skills_required" : [ "tailor", 1 ],
+  "difficulty": 1,
+  "time": 15000,
+  "reversible": false,
+  "autolearn": true,
+  "components": [
```

Comment:
Interesting, though at the moment the recipe requires soap and water, but no wool staples. o.O